### PR TITLE
Jive, and bay speech

### PR DESCRIPTION
**Pull request by NanakoAC**
I set out to port things from bay's hear_say.dm, turns out we pretty much have everything already

But in the process i decided to do something else i've been planning, so this PR also ports bay's Sign language, and redesigns it as Jive.

Directly adapted from the sign language of the same name from William Gibson's Neuromancer, Jive is a method of communication involving gestures and signals to quietly and covertly conduct illicit trade and negotiating criminal stuff.

With this culture in mind, guild and civilian roles on the ship have a pretty high chance of getting it by default.

In addition, this ports bay's Language selector (activates it really), and allows users to choose one additional language to start with at character creation. Serbian, Techno Russian and Jive are the currently available choices

### CODE REVIEW

File: code/modules/mob/living/default_language.dm [hunk 1]
```diff
@@ -23,3 +23,14 @@
...
+
+/mob/living/proc/set_default_language(var/langname)
+	var/datum/language/L
+	//Support for passing a datum directly, or the name of a language to go fetch. Very flexible proc
+	if (istype(langname, /datum/language))
+		L = langname
+	else
+		L = all_languages[langname]
+	if (!(L in languages))
+		languages.Add(L)
```
[the remark this answers is not in the document]
Comment (LudwigVonChesterfield) [reply code/modules/mob/living/default_language.dm]: languages += L even faster.

[the remark this answers is not in the document]
Comment (NanakoAC) [reply code/modules/mob/living/default_language.dm]: This is not performance critical code, so thats completely irrelevant

But also, how could |= be faster? That has to check everything to make sure the item being added doesnt already exist?

[the remark this answers is not in the document]
Comment (NanakoAC) [reply code/modules/mob/living/default_language.dm]: i'm sure dan is right, that += would be faster than .Add, but using |= seems like it'd be slower than both

File: code/modules/mob/language/language.dm [hunk 2]
```diff
@@ -233,6 +229,7 @@
 			set_default_language(null)
 		else
 			var/datum/language/L = locate(href_list["default_lang"])
+			if((L in languages))
```
[the remark this answers is not in the document]
Comment (NanakoAC) [reply code/modules/mob/language/language.dm]: will do

File: code/modules/mob/living/default_language.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (LethalGhost) [reply code/modules/mob/living/default_language.dm]: @NanakoAC ’|=’ is faster then ’if(!A in list) list += A’

**Comment by NanakoAC** [comment]
all requested changes done

**Pull request merged**